compiler/optimizer/ValuePropagationCommon.cpp

```diff
@@ -1170,6 +1170,16 @@ void OMR::ValuePropagation::transformArrayCopyCall(TR::Node *node)
          referenceArray2 = dstObject->getClassType()->isReferenceArray(comp());
          }
 
+      if (primitiveArray1 || primitiveArray2)
+         {
+         transformTheCall = primitiveTransform;
+         }
+
+      if (primitiveArray1 || primitiveArray2)
+         {
+         transformTheCall = referenceTransform;
+         }
```

Comment:
Yes I did. Fixed in 9289c54b0b26838c5753fc386b9f2dd7ce8ba98b. Thanks for catching this!